cdk/lib/national-delivery-fulfilment.ts

```diff
@@ -82,17 +82,30 @@ export class NationalDeliveryFulfilment extends GuStack {
                 "AllowFulfilmentBucketPolicy",
                 {
                     actions: [
-                        "s3:GetObject",
                         "s3:ListBucket"
                     ],
                     resources: [
-                        `arn:aws:s3:::${bucketName}/*`,
                         `arn:aws:s3:::${bucketName}`
```

Comment:
it's harder to restrict this as we need to add a condition [1], which doesn't seem to be available in gucdk

[1] https://docs.aws.amazon.com/AmazonS3/latest/userguide/amazon-s3-policy-keys.html#condition-key-bucket-ops-2